cadc-uws-server/src/main/java/ca/nrc/cadc/uws/web/JobCreator.java

```diff
@@ -112,7 +113,7 @@ else if (inlineContentHandler != null)
             log.error("failed to get request path", oops);
         }
         
-        job.setRemoteIP(request.getRemoteAddr());
+        job.setRemoteIP(NetUtil.getClientIP(request));
```

Comment:
I guess this is how the job is created in UWS sync because we're dealing with HttpServletRequest.  How is the job created in async UWS (the restlet part of UWS)?

Reply:
The Async UWS jobs are created in the `ca.nrc.cadc.uws.web.restlet.resources.AsynchResource` class, using the `ca.nrc.cadc.uws.web.restlet.resources.UWSResource.getRemoteIP()` method.